readthedocs/docsitalia/static-src/projects/css/_serp.less

```diff
@@ -6,12 +56,47 @@
   flex-direction: row;
 
   .serp-results-order {
+    position: relative;
+
     select {
-      background: white;
+      background: #fff;
       box-shadow: none;
       border: 1px solid #D8D8D8;
+      border-radius: 6px;
       color: #06c;
       text-shadow: none;
+      font-size: 16px;
+      line-height: 1.3;
+      padding: 5px 25px 5px 8px;
```

Comment:
Sempre multipli di 4px o le classi `p*-*`

Reply:
Questo è un caso particolare perché si tratta di un tipo di select non presente su Bootstrap Italia. Nel mockup viene usata la select nativa dei Mac su Chrome. È stato un po "tricky" farla uguale in tutti gli altri browser ma alla fine sono riuscito a sistemare le misure in modo che i valori del padding siano multipli di 4: `4px 32px 4px 8px;`